<launch>

<arg name = 'nchecks' default = '3' />
<arg name = 'vid_folder' default = '/cw_videos'/>

<node pkg = "second_coursework" type = 'move_robot_service.py' name = 'move_robot_service' />
<node pkg = "second_coursework" type = 'check_rooms_server.py' name = 'check_rooms_server' output = 'screen' />
<node pkg = "second_coursework" type = 'main_node.py' name = 'main_node' output = 'screen'>
    <param name = 'nchecks' value = '$(arg nchecks)' />
</node>

<include file = '/opt/itr_ws/src/hmi_robin/speech_database/launch/speech.launch' />

<node pkg = 'second_coursework' type = 'itr_cw_video.pyc' name = 'video_player' args = '$(arg vid_folder)'/>

</launch>